tests/unit/src/main/scala/tests/TestingClient.scala

```diff
@@ -66,6 +69,24 @@ final class TestingClient(workspace: AbsolutePath, buffers: Buffers)
   ): Unit = {
     clientCommands.addLast(params)
   }
+
+  override def applyEdit(
+      params: ApplyWorkspaceEditParams
+  ): CompletableFuture[ApplyWorkspaceEditResponse] = {
+    def applyEdits(uri: String, textEdits: java.util.List[TextEdit]): Unit = {
+      val path = AbsolutePath(uri)
```

Comment:
this does some additional magic to string uri (see [XtensionStringUriProtocol::toAbsolutePath](https://github.com/scalameta/metals/blob/7f85d79c3596721e3afe7b7443860c8582648c86/metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala#L329))
```suggestion
      val path = uri.toAbsolutePath
```

Reply:
After this change exception is thrown: `Missing scheme java.lang.IllegalArgumentException: Missing scheme`. It seems that `uri` is actually not a valid URI.